zerver/openapi/zulip.yaml

```diff
@@ -5265,6 +5254,131 @@ paths:
                         "scheduled_message_id": 42,
                         "result": "success",
                       }
+        "400":
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - allOf:
+                      - $ref: "#/components/schemas/NonExistingStreamIdError"
+                      - description: |
+                          A typical failed JSON response for when a stream message is scheduled
+                          to be sent to a stream that does not exist:
+                  - allOf:
+                      - $ref: "#/components/schemas/CodedError"
+                      - example:
+                          {
+                            "code": "BAD_REQUEST",
+                            "msg": "Invalid user ID 10",
+                            "result": "error",
+                          }
+                        description: |
+                          A typical failed JSON response for when a direct message is scheduled
+                          to be sent to a user that does not exist:
+  /scheduled_messages/{scheduled_message_id}:
+    patch:
+      operationId: update-scheduled-message
+      tags: ["scheduled_messages"]
+      summary: Edit a scheduled message
+      description: |
+        Edit an existing [scheduled message](/help/schedule-a-message).
+
+        **Changes**: New in Zulip 7.0 (feature level 183).
+      parameters:
+        - name: scheduled_message_id
+          in: path
+          schema:
+            type: integer
+          description: |
+            The ID of the scheduled message to update.
+
+            This is different from the unique ID that a message would have
+            after it was sent.
```

Comment:
Somehow the "was" reads oddly to me here. I think basically I read it as past tense (which then doesn't make sense), and then have to consciously see that it's actually meant as subjunctive.

If you want to keep this in the conditional ("would"), consider evading "was", like:
> would have after being sent

> would have once sent

Reply:
Yeah, the events descriptions have it in the future subjunctive text which reads much more clearly. I went with this version and updated it for both the edit and delete parameters:

> This is different from the unique ID that a message would have after being sent.